resources/windows/wix/appdatafiles.wxs

```diff
@@ -185,8 +188,8 @@
         Guid="4E7C234B-2837-4459-A20E-9503A922FCB9"
         Directory="FactsDotD">
         <CreateFolder>
-          <Permission GenericAll="yes" User="Administrators" />
-          <Permission GenericRead="yes" GenericExecute="yes" User="Everyone" />
+          <Permission GenericAll="yes" User="[WIX_ACCOUNT_ADMINISTRATORS]" />
```

Comment:
Sean - See the documentation for http://wixtoolset.org/documentation/manual/v3/customactions/osinfo.html.
This and a number of OSInfo variables are set if you put in a reference to any of them.
Unfortunately ```WIX_ACCOUNT_EVERYONE``` isn't set by this, hence the need for an additional custom action.